.github/workflows/driftctl-action.yml

```diff
@@ -24,6 +24,24 @@ jobs:
           DCTL_DRIFTIGNORE: "tests/.driftignore"
           DCTL_FILTER: "Type=='unknown_type'" # Makes driftctl to exit 0 each time
 
+  test-args:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Run driftctl
+        uses: ./
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: us-east-1
+          args: |
```

Comment:
this should be within the `with` key in the same level of `env` key.

Reply:
ugh yes, got it!